tests/test_workers/test_tasks/test_build.py

```diff
@@ -529,7 +529,7 @@ def test_skopeo_copy_fail_max_retries(mock_run_cmd):
 @mock.patch('iib.workers.tasks.build._add_label_to_index')
 @mock.patch('iib.workers.tasks.build._get_present_bundles')
 @mock.patch('iib.workers.tasks.build.set_registry_token')
-@mock.patch('iib.workers.tasks.build.is_image_dc')
+@mock.patch('iib.workers.tasks.build.is_image_fbc')
 def test_handle_add_request(
     mock_iidc,
```

Comment:
Minor: Can we rename this to mock_iifbc?

Reply:
oh I missed that, thanks!